src/api/java/com/minecolonies/api/entity/pathfinding/PathingOptions.java

```diff
@@ -43,7 +43,7 @@ public class PathingOptions
     /**
      * Cost to traverse trap doors
      */
-    public double traverseToggleAbleCost = 2D;
+    public double traverseToggleAbleCost = 10D;
```

Comment:
I would only set that for raiders I think?

Reply:
nah I think both is fine, going through doors for citizens is also rather difficult

Reply:
Yeah, but I think for citizens its nice if "they take the main door" instead of taking the back entrance for example =D

Reply:
well normally both of them have doors^^ also 10 isnt that high, its kinda check 10 blocks to the side before going here